android/modules/media/src/java/ti/modules/titanium/media/TiCameraActivity.java

```diff
@@ -47,22 +49,30 @@
 import android.view.ViewGroup.LayoutParams;
 import android.view.WindowManager;
 import android.widget.FrameLayout;
+import android.content.pm.PackageManager;
 
 @SuppressWarnings("deprecation")
-public class TiCameraActivity extends TiBaseActivity implements SurfaceHolder.Callback
+public class TiCameraActivity extends TiBaseActivity implements SurfaceHolder.Callback, MediaRecorder.OnInfoListener
 {
 	private static final String TAG = "TiCameraActivity";
 	private static Camera camera;
 	private static Size optimalPreviewSize;
+	private static Size optimalVideoSize;
 	private static List<Size> supportedPreviewSizes;
+	private static List<Size> supportedVideoSizes;
 	private static int frontCameraId = Integer.MIN_VALUE; // cache
 	private static int backCameraId = Integer.MIN_VALUE; //cache
+	private static final int VIDEO_QUALITY_LOW = CamcorderProfile.QUALITY_LOW;
+	private static final int VIDEO_QUALITY_HIGH = CamcorderProfile.QUALITY_HIGH;
+	private static final String MEDIA_TYPE_PHOTO = "public.image";
+	private static final String MEDIA_TYPE_VIDEO = "public.video";
+	
 	private TiViewProxy localOverlayProxy = null;
 	private SurfaceView preview;
 	private PreviewLayout previewLayout;
 	private FrameLayout cameraLayout;
 	private boolean previewRunning = false;
-	private int currentRotation;
+	private static int currentRotation;
```

Comment:
you're right. I had a static function before and the compiler complaint that I'm using a non static variable. The function got removed so it doesn't need to be static anymore.